src/civibuild.lib.sh

```diff
@@ -645,6 +645,9 @@ function civicrm_enable_riverlea_theme() {
   if civicrm_check_ver '>' 5.80.alpha1 ; then
     cv en --ignore-missing riverlea
     cv vset theme_backend=minetta
+    if [[ "$CIVI_UF" == "Drupal" ] || [ "$CIVI_UF" == "WordPress" ]]; then
```

Comment:
I needed to change to this:

```diff
diff --git a/src/civibuild.lib.sh b/src/civibuild.lib.sh
index 8ccd4fb..02a08e5 100644
--- a/src/civibuild.lib.sh
+++ b/src/civibuild.lib.sh
@@ -645,7 +645,9 @@ function civicrm_enable_riverlea_theme() {
   if civicrm_check_ver '>' 5.80.alpha1 ; then
     cv en --ignore-missing riverlea
     cv vset theme_backend=minetta
-    if [[ "$CIVI_UF" == "Drupal" ] || [ "$CIVI_UF" == "WordPress" ]]; then
+    if [ "$CIVI_UF" == "Drupal" ] ; then
+       cv vset riverlea_dark_mode_backend=light
+    elif [ "$CIVI_UF" == "WordPress" ] ;  then
       cv vset riverlea_dark_mode_backend=light
     fi
   fi
```

I was getting errors using the `||` syntax

Reply:
with this setup - the back end setting is updated properly 

should we also set front end here?